language/diem-framework/modules/AccountFreezing.move

```diff
@@ -9,26 +9,26 @@ module AccountFreezing {
     use 0x1::CoreAddresses;
     use 0x1::Roles;
 
-    struct FreezingBit has key, store {
+    struct FreezingBit has key {
         /// If `is_frozen` is set true, the account cannot be used to send transactions or receive funds
         is_frozen: bool,
     }
 
-    struct FreezeEventsHolder has key, store {
+    struct FreezeEventsHolder has key {
         freeze_event_handle: EventHandle<FreezeAccountEvent>,
         unfreeze_event_handle: EventHandle<UnfreezeAccountEvent>,
     }
 
     /// Message for freeze account events
-    struct FreezeAccountEvent has copy, drop, store {
+    struct FreezeAccountEvent has drop, store {
```

Comment:
Should events have `store`? I don't think it's harmful, but I also don't think we currently store any of them + it would be a non-breaking change to add `store` later if we change our minds.

Reply:
Meta: unused ability compiler warning would be fun future work :)